lib/std/os/windows/kernel32.zig

```diff
@@ -82,6 +82,8 @@ pub extern "kernel32" stdcallcc fn GetFileAttributesW(lpFileName: [*]const WCHAR
 
 pub extern "kernel32" stdcallcc fn GetModuleFileNameW(hModule: ?HMODULE, lpFilename: [*]u16, nSize: DWORD) DWORD;
 
+pub extern "kernel32" stdcallcc fn GetModuleHandleA(lpModuleName: ?[*]const CHAR) ?HMODULE;
```

Comment:
Can you use the `GetModuleHandleW` function instead? We've been trying to move away from the `A` versions (see #534) because they depend on globally configurable registry settings. Same thing with `LoadLibraryA` below.